import-export-cli/mi/impl/common.go

```diff
@@ -237,3 +238,12 @@ func createErrorWithResponseBody(resp string, err error) error {
 	}
 	return err
 }
+
+func updateArtifactState(url, artifactName, state, env string) (string, error) {
+	body := dedent.Dedent(`{
+		"name": "` + artifactName + `",
+		"status": "` + state + `"
+	}`)
```

Comment:
Any reason for not using a struct instead? We can avoid needing to depend on the third party package `dedent`

Reply:
Fixed with a0db0eb. Did some refactoring to `InvokePOSTRequest` to use interfaces so we can pass strings, structs, maps or byte arrays directly as the body